rsapi/commands.go

```diff
@@ -104,6 +105,12 @@ func (a *API) ParseCommand(cmd, hrefPrefix string, values ActionCommands) (*Pars
 					name, value)
 			}
 			*coerced = append(*coerced, APIParams{name: val})
+		case "*time.Time":
+			val, err := time.Parse(time.RFC3339, value)
```

Comment:
Would be really nice to support a Unix timestamp integer (seconds since epoch)...